.gitlab-ci.yml

```diff
@@ -29,22 +29,56 @@ variables:
     paths:
       - artifacts/
 
-test-linux:
+test-audit:
   stage:                           test
-  variables:
-    RUN_TESTS:                     all
   script:
-    - scripts/gitlab/test-all.sh
+    - set -e
+    - set -u
```

Comment:
Since `cargo audit` is not a script these commands will have no effect.

Reply:
indeed, thanks